coralproject/talk
# [CORL-1641/42] Disable rating edit for R&R, tab count fixes

## What does this PR do?

- Disable editing a rating only comment for R&R, can still edit if it's a review comment
- Tab counts will update for everything except the Review tab
  - Why? Cause the review tab improperly re-uses the `AllCommentsTab` and simply sets a `tag` to filter the comments. This works for loading the comments but makes for a bunch of extra logic that will need to be added to manually update counts in Relay for both the `CreateComment` and `CommentEnteredSubscription`. If we created another comment stream, it would work better, but alas, it is what it is now.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Create a R&R stream
- Go to it

Edit:
- Create a rating only comment
- Quickly go to comment history
- Find rating only comment
- Single conversation view on it
- Edit button should be missing

Counts:
- Post as one commenter while watching with another commenter
- Switch the tabs around and make sure the counts update when another commenter posts to different comment tab than the one you are on
